### DISCUSSION TITLE
Sorting in descending order of created date

### DISCUSSION THREAD

**Discussion by team-wealthian** [VirtusLab-Open-Source/strapi-plugin-comments#106] [Q&A]
@cyp3rius Can you please confirm if there exists any way we can sort the data based on  crated date field in descending order using graph ql

**Comment by cyp3rius** [comment]
Hello @team-wealthian,

it wasn't possible for client endpoints / client GQL but I've made a change already :) It is going to be part of upcoming release. 

`findAll` is going to support:
- filtering
- sorting
- pagination

`findAllInHierarchy` is going to support only sorting because of hierarchical nature and possibility to break / fake it by results manipulation.

**Comment by cyp3rius** [comment]
Released as part of `v2.0.2`
